src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -29,17 +32,160 @@ describe.skip('Alerts (Unit)', () => {
       .compile();
 
     app = moduleFixture.createNestApplication();
+    loggingService = moduleFixture.get(LoggingService);
+
     await app.init();
   });
 
   afterAll(async () => {
     await app.close();
   });
 
-  it('returns 200 (OK) for valid payload', async () => {
-    const alert = alertBuilder().build();
+  it('notifies about addOwnerWithThreshold attempts', async () => {
+    const consoleSpy = jest.spyOn(console, 'log');
```

Comment:
The `console` usage here is supposed to be work in progress right?

If so I would not create a test for it as it will ultimately mean that the tests need to be changed.

Reply:
This was a pre-emptive test but I've instead made it `todo` in 0d89ce993ba48efe4bc94ea60ce9bca006f5c226.

Reply:
Same in "handles alerts with multiple logs" test.

Overall I don't think we should test WIP as the logs themselves are subject to change.

Reply:
Ok, I'll keep that in mind. I've marked the other test in question as `todo` in 0f3996a20c020fa2c781fa25c1ee563c3ec08b40.